models/demos/falcon7b_common/tests/perplexity/test_perplexity_falcon.py

```diff
@@ -36,27 +37,25 @@ def test_perplexity(
     expected_ppl,
     expected_top1,
     expected_top5,
-    async_mode,
+    enable_async_mode,
```

Comment:
Is enable_async_mode used?

Reply:
Yes, it is a fixture in conftest that enables async: https://github.com/tenstorrent/tt-metal/blob/main/conftest.py#L325

Reply:
Nvm, I just saw it's a fixture.